sdk/provisioning/Azure.Provisioning.CloudMachine/src/Client/MessagingServices.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.CloudMachine;
+
+public static class MessagingServices
+{
+    public static void Send(this CloudMachineClient cm, object serializable)
+    {
+        ServiceBusSender sender = cm.ClientCache.Get<ServiceBusSender>("cm_default_topic", () =>
+        {
+            ServiceBusClient sb = new(cm.Properties.ServiceBusNamespace, cm.Credential);
+            ServiceBusSender sender = sb.CreateSender("cm_default_topic");
+            return sender;
```

Comment:
Maybe we could reuse part of the `IClientCreator<TClient, TOptions>` interface?

Reply:
I added a comment to consider.